sopel/modules/emoticons.py

```diff
@@ -8,104 +8,108 @@
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from sopel.module import commands, example
+from sopel import module
 
 
-@commands('shrug')
-@example('.shrug', r'¯\_(ツ)_/¯')
+@module.commands('shrug')
+@module.action_commands('shrugs')
+@module.example('.shrug', r'¯\_(ツ)_/¯')
 def shrug(bot, trigger):
     bot.say('¯\\_(ツ)_/¯')
 
 
-@commands('happy')
-@example('.happy', 'ᕕ( ᐛ )ᕗ')
+@module.commands('happy')
+@module.example('.happy', 'ᕕ( ᐛ )ᕗ')
 def happy(bot, trigger):
     bot.say('ᕕ( ᐛ )ᕗ')
 
 
-@commands('tableflip', 'tflip')
-@example('.tableflip', '(╯°□°）╯︵ ┻━┻')
-@example('.tflip', '(╯°□°）╯︵ ┻━┻')
+@module.commands('tableflip', 'tflip')
+@module.action_commands('flips table')
```

Comment:
If we're really planning to remove regex support from all command types, gotta do this:

```suggestion
@module.action_commands('flips table', 'flips a table', 'flips the table')
```

If we can keep regex for at least action commands, and maybe nickname commands (side issue):

```suggestion
@module.action_commands('flips (?:(?:a|the) )?table')
```

I didn't think of stuff like this when I just went along with the idea that regex in commands is too confusing. I still agree that for plain old `@commands`, it is, but this case shows how the non-prefixed command types could still benefit from keeping regex support.

This definitely belongs in a separate issue; I'm just taking advantage of this specific case (and the `unflips` one below) to make my argument for me, before anything gets opened. :D

Reply:
On the other hand, I'm still not convinced that `action_commands` is the right name for it, as they act more like generic rules. So...